src/js/lib/content/src/docsStyle.less

```diff
@@ -0,0 +1,325 @@
+@import "https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@300;400;500;600;700&display=swap";
+
+.doc {
+  font-family: DejaVuSansMonoBook, sans-serif;
```

Comment:
Is this font what is used in the reference style? I seem to remember that was `Causten`

Reply:
Till now we've used dejavu